osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -729,51 +734,51 @@ private void updateYPositions()
         {
             visibleItems.Clear();
 
-            float currentY = visibleHalfHeight;
+            // visibleHalfHeight includes BleedTop, but Scroll screenspace doesn't, so we remove it here
+            float currentY = visibleHalfHeight - BleedTop;
 
             scrollTarget = null;
 
-            foreach (CarouselItem item in root.Children)
+            double elapsed = Clock.CurrentTime - timeSinceLastYUpdate;
```

Comment:
this reads bogus. if this intends to be a time delta, it should be `timeOfLastYUpdate`, not `timeSinceLastYUpdate`.